doc/news/changes/minor/20230508tjhei

```diff
@@ -0,0 +1,5 @@
+Changed: MPI communicators (of type `MPI_Comm`) are not treated as
+trivially copyable POD types throughout the library. This means that
+we store and pass them by value and by reference.
```

Comment:
and here
```suggestion
we store and pass them by value instead of by reference.
```
?

Reply:
somebody was clearly half asleep when he wrote these two sentences in the changelog entry. :-)

Reply:
Certainly. 😃